### DISCUSSION TITLE
Better config to create image in Playground like Playground.com

### DISCUSSION THREAD

**Discussion by thiagovargas** [lllyasviel/Fooocus#3554] [Q&A]
Hello, im trying to generate images using playground models, but the quality is not so good like in playground.com
Im getting too a strange hands e some times strange foots, big arms, etc...

How can i increase my image generation?

Thank you

**Comment by DavidDragonsage** [comment]
I have never used playground.com but is does seem to have higher quality imagery than what we typically achieve with the open source Playground version. I like Playground in Fooocus as an alternative model, but as far as general quality it is the lowest rated model that I have tested. It makes me wonder if the open source release is not really the same as the one used at playground.com.

You will find extensive coverage of Playground at the Pure Fooocus Facebook group, in the Guide section. There is one Guide devoted to Playground and it is also one of the models in the Base Model Reviews Guide.
